modules/es-extensions/store/extensions/app/greg-diff-api/app.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Thanks @thushara35 , I have updated the copyright year in [this commit](https://github.com/wso2/product-greg/pull/854/commits/1ff96980dd4b07ff33d878458c4ef1722c6871df)